pinot-core/src/test/java/org/apache/pinot/queries/InterSegmentOrderByMultiValueQueriesTest.java

```diff
@@ -38,10 +38,7 @@
   @Test(dataProvider = "orderByDataProvider")
   public void testGroupByOrderByMVSQLResults(String query, List<Object[]> expectedResults,
       long expectedNumEntriesScannedPostFilter, DataSchema expectedDataSchema) {
-    Map<String, String> queryOptions = new HashMap<>(2);
```

Comment:
I would recommend to not change existing unit tests to accommodate new tests. It will reduce the coverage like in this case we were testing out PQL execution path and we changed to test SQL. 

We should simply add a new unit test.

Reply:
all other tests use either full pql or full sql. this middle ground was added by me when we were not ready.i dont see why we should keep this half approach, now that we have the new endpoint ready